deps/pom.xml

```diff
@@ -47,16 +47,6 @@
             <artifactId>qpid-proton-j-extensions</artifactId>
             <version>1.2.3</version>
         </dependency>
-        <dependency>
-            <groupId>org.bouncycastle</groupId>
-            <artifactId>bcmail-jdk15on</artifactId>
-            <version>1.69</version>
-        </dependency>
-        <dependency>
-            <groupId>org.bouncycastle</groupId>
```

Comment:
We only had this bouncycastle dependency for parsing private keys, and we don't do that anymore